services/jenkins/jenkins-coverage-redirector.service.js

```diff
@@ -0,0 +1,18 @@
+'use strict'
+
+const { redirector } = require('..')
+
+module.exports = [
```

Comment:
Since there's only one, you can `module.exports = redirector({ ... })` and then you can use `createServiceTester()`.

Reply:
Nice! For some reason I had it in my mind that `createServiceTester` couldn't be used for redirects but that's for deprecated services

Reply:
Done!